osm2gtfs/creators/stops_creator.py

```diff
@@ -48,14 +48,18 @@ def remove_unused_stops_from_feed(self, feed):
         from the final GTFS.
         It is called after the whole GTFS creation inside the main program.
         """
-        removed = 0
+        unused_stops = []
         for stop_id, stop in feed.stops.items():
             if stop.location_type == 0 and not stop.GetTrips(feed):
-                removed += 1
-                del feed.stops[stop_id]
-        if removed == 0:
+                #del feed.stops[stop_id]
+                unused_stops.append(stop_id)
+        if len(unused_stops) == 0:
```

Comment:
These two lines don't seem useful :thinking: 
```
        if len(unused_stops) == 0:
            pass
```

Reply:
That `if` was there before this PR. But I'll delete it ( and in  `routes_creator.py`), I agree with you.